File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/SimpleFeature.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SimpleFeature {

    @SerializedName("xid")
    @Expose
    private String xid;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("rate")
    @Expose
    private Integer rate;
    @SerializedName("osm")
    @Expose
    private String osm;
    @SerializedName("wikidata")
    @Expose
    private String wikidata;
    @SerializedName("kinds")
    @Expose
    private String kinds;
    @SerializedName("point")
    @Expose
    private Point point;

    public String getXid() {
        return xid;
    }

    public void setXid(String xid) {
        this.xid = xid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getRate() {
        return rate;
    }

    public void setRate(Integer rate) {
        this.rate = rate;
    }

    public String getOsm() {
        return osm;
    }

    public void setOsm(String osm) {
        this.osm = osm;
    }

    public String getWikidata() {
        return wikidata;
    }

    public void setWikidata(String wikidata) {
        this.wikidata = wikidata;
    }

    public String getKinds() {
        return kinds;
    }

    public void setKinds(String kinds) {
        this.kinds = kinds;
    }

    public Point getPoint() {
        return point;
    }

    public void setPoint(Point point) {
        this.point = point;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/SimpleFeature.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class SimpleFeature {
    @SerializedName("xid")
    @Expose
    private var xid: String? = null

    @SerializedName("name")
    @Expose
    private var name: String? = null

    @SerializedName("rate")
    @Expose
    private var rate: Int? = null

    @SerializedName("osm")
    @Expose
    private var osm: String? = null

    @SerializedName("wikidata")
    @Expose
    private var wikidata: String? = null

    @SerializedName("kinds")
    @Expose
    private var kinds: String? = null

    @SerializedName("point")
    @Expose
    private var point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? = null
    fun getXid(): String? {
        return xid
    }

    fun setXid(xid: String?) {
        this.xid = xid
    }

    fun getName(): String? {
        return name
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getRate(): Int? {
        return rate
    }

    fun setRate(rate: Int?) {
        this.rate = rate
    }

    fun getOsm(): String? {
        return osm
    }

    fun setOsm(osm: String?) {
        this.osm = osm
    }

    fun getWikidata(): String? {
        return wikidata
    }

    fun setWikidata(wikidata: String?) {
        this.wikidata = wikidata
    }

    fun getKinds(): String? {
        return kinds
    }

    fun setKinds(kinds: String?) {
        this.kinds = kinds
    }

    fun getPoint(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? {
        return point
    }

    fun setPoint(point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point?) {
        this.point = point
    }
}